### ISSUE TITLE
Senior Technical Editor

### ISSUE THREAD

**Issue by annlou7** [MicrosoftDocs/microsoft-style-guide#3]
I like this section on acronyms and plan to share it with the writers on my team. However, in the first sentence, despite the guidelines, SEO was not spelled out. I suggest this edit to be consistent with the content in this section: Acronyms and abbreviations can have an adverse effect on clarity, voice, and search engine optimization (SEO). We can't assume all readers know what SEO means.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0d30bb26-5486-e2d9-851b-47ceeb6d07b5
* Version Independent ID: 295d184a-a254-acea-6185-b14346fb0634
* Content: [Acronyms - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/acronyms)
* Content Source: [styleguide/acronyms.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/acronyms.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**

**Comment by camind** [comment]
Thanks for your comment! We're glad you find the section useful. 
As for not spelling out "SEO": the key is that the sentence says "the abbreviations *can* have an adverse effect on clarity" and so on, not that they *will* have an adverse effect.

Some abbreviations and acronyms are as well known--if not more so--as their spelled-out form. This page also has the guideline, "If you’re sure your audience is familiar with an acronym, it’s OK to use it without spelling it out." 
*SEO* is in the American Heritage Dictionary, which indicates that it's become common usage in the language. The style guide editors figured that the people reading this style guide--who are writing about computer technology--would be familiar with the acronym.

**Comment by annlou7** [comment]
Hi Catherine:

Thanks for your reply. I appreciated it. Currently I’m a senior technical editor in the area of  fiber-optic hardware and software. I had to look up SEO because I’ve mostly worked for companies like Cisco and Polycom where we did not write much about web searches in our documentation. So for me, SEO was not something I automatically knew. But to your point, each reader is different as to what they already know in relation to acronyms and abbreviations.

This week I sent the page on Acronyms from the MS Writing Style Guide to my group of writers as a reminder for how to handle these. ☺

Best,
Ann Arnold
